distributed/deploy/ssh2.py

```diff
@@ -68,13 +70,15 @@ def __init__(
         self,
         scheduler: str,
         address: str,
+        worker_module: str,
```

Comment:
Not sure if anyone relies on this class directly, but if it's a possibility, maybe we should make `"distributed.cli.dask_worker"` the default value?

Reply:
Good point. It is the default up in the `SSHCluster` class, and I assumed that the `Worker` class would never be used anywhere else. But maybe that's a bad assumption?

Reply:
I don't know if it's a bad assumption, maybe nobody uses it, but since it's easy to let the current behavior unchanged, probably better to play it safe.

Reply:
I think it makes sense to set `worker_module="distributed.cli.dask_worker"` as the default value.  This is probably slightly nicer for people looking at the docstring.  It gives them a sense for what kind of value they should place here.

Reply:
Sure! In which class `SSHCluster` or `Worker` or both?

Reply:
I would add the default in `SSHCluster` because that is the most visible.  I think of the `Worker` class as being mostly an implementation detail.  It could be added there though.  Adding it in both places is probably the most safe option.